version/version.go

```diff
@@ -9,8 +9,8 @@ import (
 const validCharacters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz-"
 
 const (
-	appMajor uint = 1
-	appMinor uint = 1
+	appMajor uint = 2
```

Comment:
Ok, we will have a `mainnet` release with `1.2.0` in couple of days as well. Well aligned.